cameraview/src/main/views/com/otaliastudios/cameraview/CameraPreview.java

```diff
@@ -41,9 +41,16 @@
     protected int mInputStreamHeight;
     protected boolean mInputFlipped;
 
+    private final boolean mHasOverlay;
```

Comment:
As said, this is not needed anymore in any CameraPreview class. So you can leave previews unchanged.